ee_tests/src/quickstart_deployments.spec.ts

```diff
@@ -53,13 +53,13 @@ describe('Main E2E test suite', () => {
     }
   });
 
-  it('Login', async () => {
+  it('login', async () => {
     support.info('--- Login ---');
     let login = new support.LoginInteraction();
     await login.run();
   });
 
-  it('Check feature level', async () => {
+  it('feature-level', async () => {
```

Comment:
I think that underscores make the string more readable but you can leave it like this